CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### enhancement
 - Create E2E resources Helm chart by @juanjjaramillo in [#377](https://github.com/newrelic/newrelic-infra-operator/pull/377)
 - Create E2E tests by @juanjjaramillo in [#378](https://github.com/newrelic/newrelic-infra-operator/pull/378)
+- Create E2E workflow by @juanjjaramillo in [#379](https://github.com/newrelic/newrelic-infra-operator/pull/379)
```

Comment:
Can you please move this to the ## Unreleased heading?

Reply:
Nice catch! Done